### DISCUSSION TITLE
Add ability to record session

### DISCUSSION THREAD

**Discussion by Neophen** [keathley/wallaby#780] [Ideas]
I have a need to record a video to preview, at the moment the way i do this:
- add pause in between steps
- set resolution in capabilities
- launch quicktime
- launch tests
- trim videos

But apparently it is possible through chrome tools

https://medium.com/@anchen.li/how-to-do-video-recording-on-headless-chrome-966e10b1221

If you ever have the time it would be amazing,
I could try creating a PR also for something

```
session
|> Browser.start_recording(width,height)
|> do_tests()
|> Browser.end_recording_and_save("path/to/file")
```

Any pointers would be amazing!

**Comment by mhanberg** [comment]
Unfortunately cypress uses the chrome devils protocol to control chrome whereas Wallaby uses chrome driver, and as far as I know it doesn't provide this capability. 

I think there are elixir bindings for CDP tho, so you could explore that avenue. 

I have plans to add a CDP adapter for wallaby, but I haven't had the time yet.

**Reply by shekohex** [reply]
Hey, sorry to bring this up again, is that would be possible today?
